_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Contributing a Blog Post to pulpproject.org
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `YYYY-MM-DD-title-of-post.md` e.g.
+   `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+1. Commit your changes to the `gh-pages` branch.
```

Comment:
I recommend adding a sentence here to explain why we deviate from what is normal.

```
By committing directly onto the `gh-pages` branch, you can preview it using Github.
```